inventory/upcloud.py

```diff
@@ -85,21 +89,24 @@
     import simplejson as json
 
 
-def get_hostname_or_ip(server, get_ip_address=False, get_non_fqdn_name=False):
+def get_hostname_or_ip(server, get_ip_address, get_non_fqdn_name, addr_family):
```

Comment:
Function was refactored to not use optional arguments since there is no precedent where optional arguments are not passed.